new-log-viewer/src/typings/worker.ts

```diff
@@ -41,13 +41,15 @@ enum WORKER_REQ_CODE {
     EXPORT_LOG = "exportLog",
     LOAD_FILE = "loadFile",
     LOAD_PAGE = "loadPage",
+    QUERY_LOG = "queryLog",
```

Comment:
Shall we name this request code as `SUBMIT_QUERY` to reflect the behaviour?

Reply:
@kirkrodrigues What do you think? (Could you also help review the protocol changes in this file? )

Reply:
how about START_QUERY? It looks more straightforward to me